packages/pyright-internal/src/analyzer/program.ts

```diff
@@ -240,6 +240,9 @@ export class Program {
                 const filePath = normalizePathCase(this._fs, oldFile.sourceFile.getFilePath());
                 if (!newFileMap.has(filePath)) {
                     oldFile.isTracked = false;
+                } else {
+                    // Otherwise make sure this file is marked as tracked.
```

Comment:
Noticed this potential problem when trying to track down why none of the files in my workspace were tracked.

The root cause of my problem was that 'pyvenv.cfg' was in my root. Added this issue to maybe add a warning for this case:

https://github.com/microsoft/pylance-release/issues/3706

Reply:
I don't understand why this is necessary. This should be handled by the subsequent call to `addTrackedFiles`. In any case, this change is unrelated to the bug at hand, so it would be better to make as a separate commit.